flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/OperatorEventDispatcherImpl.java

```diff
@@ -83,41 +118,98 @@ public void registerEventHandler(OperatorID operator, OperatorEventHandler handl
         }
     }
 
-    Set<OperatorID> getRegisteredOperators() {
-        return handlers.keySet();
+    boolean containsOperatorEventGateway(OperatorID operatorId) {
```

Comment:
Would it be simpler to remove this method and put the check the above three methods?

We can rename these methods as either `maybeXXX` or `XXXIfExists`.